pkg/internal/checkup/trex/client_test.go

```diff
@@ -0,0 +1,277 @@
+				MTxExpectedBps:          21.0,
+				MTxExpectedCps:          22.0,
+				MTxExpectedPps:          23.0,
+				MTxPps:                  7988831.0,
+			},
+		}
+
+		assert.Equal(t, expected, stats, "GetGlobalStats returned unexpected result")
+	})
+}
+
+const (
+	portStatsCmd    = "cd /opt/trex && echo \"verbose on;stats --port 0 -p\" | ./trex-console -q\n"
+	portStatsOutput = "Using 'python3' as Python interpeter\r\n\r\n\r\n-=TRex Console v3.0=-\r\n\r\nType 'help' or '?' for supported act" +
+		"ions\r\n\r\ntrex>\r\n\x1b[1m\x1b[32mverbose set to on\x1b[39m\x1b[22m\r\n\r\n\r\n\r\n[verbose] Sending Request To Server:\r\n\r" +
+		"\n[\r\n    {\r\n        \"id\": \x1b[31m\"razdt1qe\"\x1b[0m,\r\n        \"jsonrpc\": \x1b[31m\"2.0\"\x1b[0m,\r\n        \"metho" +
+		"d\": \x1b[31m\"get_port_stats\"\x1b[0m,\r\n        \"params\": {\r\n            \"api_h\": \x1b[31m\"hu7wm7qq\"\x1b[0m,\r\n    " +
+		"        \"port_id\": 0\r\n        }\r\n    }\r\n]\r\n\r\n\r\n\r\n" +
+		"[verbose] Server Response:\r\n\r\n" +
+		"{\r\n" +
+		"    \"id\": \x1b[31m\"razdt1qe\"\x1b[0m,\r\n" +
+		"    \"jsonrpc\": \x1b[31m\"2.0\"\x1b[0m,\r\n" +
+		"    \"result\": {\r\n" +
+		"        \"ibytes\": 68625,\r\n" +
+		"        \"ierrors\": 10,\r\n" +
+		"        \"ipackets\": 893,\r\n" +
+		"        \"m_cpu_util\": 11.0,\r\n" +
+		"        \"m_total_rx_bps\": 6630.8935546875,\r\n" +
+		"        \"m_total_rx_pps\": 9.337542533874512,\r\n" +
+		"        \"m_total_tx_bps\": \x1b[94m1820482048\x1b[0m.0,\r\n" +
+		"        \"m_total_tx_pps\": \x1b[94m3346474\x1b[0m.25,\r\n" +
+		"        \"obytes\": \x1b[94m32640000000,\x1b[0m\r\n" +
+		"        \"oerrors\": 15,\r\n" +
+		"        \"opackets\": \x1b[94m480000000\r\n" +
+		"\x1b[0m    }" +
+		"\r\n}" +
+		"\r\n\r\n\x1b[4m\x1b[36mPort Statistics\x1b[39m\x1b[24m\r\n\r\n   port    |         0         \r\n-----------+------------------" +
+		"\r\nowner      |              \x1b[32mroot\x1b[39m \r\nlink       |                UP \r\nstate      |              \x1b[1mIDLE" +
+		"\x1b[22m \r\nspeed      |           10 Gb/s \r\nCPU util.  |              \x1b[32m0.0\x1b[39m% \r\n--         |                " +
+		"   \r\nTx bps L2  |         1.82 Gbps \r\nTx bps L1  |         2.36 Gbps \r\nTx pps     |         3.35 Mpps \r\nLine Util. |   " +
+		"        \x1b[1m23.56 %\x1b[22m \r\n---        |                   \r\nRx bps     |             0 bps \r\nRx pps     |          " +
+		"   0 pps \r\n----       |                   \r\nopackets   |                 0 \r\nipackets   |                 0 \r\nobytes   " +
+		"  |                 0 \r\nibytes     |                 0 \r\ntx-pkts    |            0 pkts \r\nrx-pkts    |            0 pkts " +
+		"\r\ntx-bytes   |               0 B \r\nrx-bytes   |               0 B \r\n-----      |                   \r\noerrors    |      " +
+		"           \x1b[32m0\x1b[39m \r\nierrors    |                 \x1b[32m0\x1b[39m \r\n\r\ntrex>Shutting down RPC client\r\n\r\n[r" +
+		"oot@dpdk-traffic-gen-jscpt trex]# "
```

Comment:
Please consider using a raw string, and/or break the lines after `\r\n`

Reply:
I prefer not to. The part that is relevant is the json part (starts with `Server Response` and ends  with the json end). This part is broken appropriately for better readability.

Regarding the rest - we don'r really care about it so we just make sure the line is not too long and break it every set characters.